queries.sql

```diff
@@ -8,3 +8,42 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
 SELECT * FROM animals WHERE wight_kg >=10.4 AND wight_kg <=17.3;
 
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+COMMIT;
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+SELECT * FROM animals;
+UPDATE animals SET species = 'pikemon' WHERE name NOT LIKE '%mon';
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE name NOT  LIKE '%mon';
+SELECT * FROM animals;
+COMMIT;
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK; 
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022/01/01';
+DELETE 1
```

Comment:
- [ ] You have added an extra line 36, kindly delete it.

Reply:
Corrected with big thanks!!